bokehjs/src/lib/models/formatters/log_tick_formatter.ts

```diff
@@ -46,10 +48,18 @@ export class LogTickFormatter extends TickFormatter {
     if (expos == null)
       return this.basic_formatter.format_graphics(ticks, opts)
     else {
+      let _low = (this.min_exponent.constructor === Array) ? min(...this.min_exponent) : -this.min_exponent
+      let _high = (this.min_exponent.constructor === Array) ? max(...this.min_exponent) : this.min_exponent
```

Comment:
All these `let` can be `const` I think, also would prefer `low` and `high` (without leading underscore)

Reply:
I will edit it. Why do you prefere variables without underline?

Reply:
@mosc9575 Also, we have some utility predicates e.g. `isArray` those should be used instead of `this.min_exponent.constructor === Array` etc.

Reply:
> Why do you prefere variables without underline?

Personally? I just don't see any use for "private" variable inside a function, where all the  variables are local to begin with. It just makes things harder to read.

Apart from that, it's just what is already established in this codebase, and we try to keep things consistent.